common/utils/utils.go

```diff
@@ -15,3 +20,21 @@ func SafeErrStr(err error) string {
 
 	return ""
 }
+
+// GetExternalIPAddress returns external IP address
+func GetExternalIPAddress() (externalIP string, err error) {
+
+	resp, err := http.Get("http://ipinfo.io/ip")
+	if err != nil {
+		return "", err
+	}
+
+	defer resp.Body.Close()
+
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return "", err
+	}
+
+	return string(body), nil
```

Comment:
it better to validate to ensure returned value is correct ip.

Reply:
@nguyen202 its tried and tested method. how do you suggest to validate?